grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -328,6 +328,10 @@ class Location implements Comparable<Location>, java.io.Serializable {
         return !onHold
     }
 
+    Boolean hasNoManagedInventoryAndSupportsSubmittingRequests() {
```

Comment:
My only comment is that this method name does not roll off the tongue. I'll try to think of a way to simplify it.

Reply:
Changed to `isDownstreamConsumer`